docs/status/branches.json

```diff
@@ -16,7 +16,7 @@
         }
     },
     "branch_actions": {
-        "cylc/cylc-flow": ["test_fast", "test_functional", "build", "test_conda-build"],
+        "cylc/cylc-flow": ["test_fast", "test_functional", "bash", "test_manylinux", "test_conda-build", "test_tutorial_workflow"],
```

Comment:
"build" removed purposefully?

Reply:
There isn't a `build.yml` workflow

Reply:
MB, the workflow was added on a PR which didn't get merged so it showed in GH but wasn't on master. I've cherry-picked that one onto master so it's there again now.

Reply:
Yeah I saw, added back in after rebase